CHANGELOG/CHANGELOG-3.0.md

```diff
@@ -0,0 +1,387 @@
+# Release notes for v3.0.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.1.0
+
+## Breaking Changes
+
+- Update volume snapshot CRDs to apiextensions.k8s.io/v1 version. ([#367](https://github.com/kubernetes-csi/external-snapshotter/pull/367), [@prateekpandey14](https://github.com/prateekpandey14))
```

Comment:
Is this breaking? no fields changed right?

Reply:
Yes, fields didn't change.  I think it depends on who are using these manifests.  For general user, this is not a breaking change as it will not break their yaml files.  For people who need to install these manifests, it is a breaking change as it moves from apiextensions.k8s.io/v1beta1 to apiextensions.k8s.io/v1.

Reply:
Why is that a breaking change? v1 is backwards compatible with v1beta1.

Reply:
Their Helm install failed because they use v1 while our manifest used v1beta1.

Anyway, I moved this to the other notable changes section.

Reply:
Addressed your comments.  PTAL.